doc/api/util.md

```diff
@@ -2004,6 +2004,23 @@ useful for addon developers who prefer to do type checking in JavaScript.
 
 The API is accessible via `require('node:util').types` or `require('node:util/types')`.
 
+### `util.types.isArrayBufferDetached(value)`
```

Comment:
a) I think this list is alphabetical

b) Should this really be a public API? Why is checking whether `new TypedArray(buffer)` throws insufficient?

Reply:
I fixed the alphabetical issue.